msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -243,19 +235,19 @@ You must supply the user ID and the tenant ID. If the call succeeds, the API ret
 
 ## References
 
-The official proactive messaging samples are listed below.
+The official proactive messaging samples are as follows:
```

Comment:
@KirtiPereira and @Mahesh2516,

Please update the sample code part. Change the heading "References"